### ISSUE TITLE
CSS: Missing UA stylesheet features

### ISSUE THREAD

**Issue by AtkinsSJ** [LadybirdBrowser/ladybird#930]
Our UA stylesheets ([here](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/CSS/Default.css), [here](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/MathML/Default.css) and [here](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/SVG/Default.css)) are mostly copied from web specs, and so contain features we do not support yet.

They're not especially more important than any other CSS features, but it would be nice if opening `about:blank` didn't give us any CSS parsing errors.

Here is a list of them:

- [ ] The `:popover-open` pseudo-class - [spec](https://html.spec.whatwg.org/multipage/semantics-other.html#selector-popover-open)
- [ ] The `unicode-bidi` property - [spec](https://www.w3.org/TR/css-writing-modes-3/#propdef-unicode-bidi)
- [ ] The `match-parent` value for the `text-align` property - [spec](https://www.w3.org/TR/css-text-3/#valdef-text-align-match-parent)
- [ ] The `min-content` value for the `min-inline-size` property - [spec](https://www.w3.org/TR/css-sizing-3/#valdef-width-min-content)
- [ ] The `left` and `right` values for the `justify-self` property - [spec](https://www.w3.org/TR/css-align-3/#propdef-justify-self)
- [ ] The `filter` property - [spec](https://www.w3.org/TR/filter-effects-1/#propdef-filter)
- [ ] The `writing-mode` property - [spec](https://www.w3.org/TR/css-writing-modes-3/#propdef-writing-mode)
- [ ] The `text-space-collapse` property - should be renamed `white-space-collapse`, [spec](https://www.w3.org/TR/css-text-4/#propdef-white-space-collapse)

**Comment by lpas** [comment]
Is this still an issue? They seem to be at least all implemented for parsing.

**Comment by AtkinsSJ** [comment]
Thanks. Looks like we don't actually do anything with our computed `unicode-bidi` values, but I've confirmed that we do implement the other things.

**Comment by ahmedrowaihi** [comment]
is this crossing any of this issue ? #7288 ?

**Comment by AtkinsSJ** [comment]
Yeah that's more specific. I'll close this now as it's no longer useful.
